### DISCUSSION TITLE
Drawing bounding box on image in ToolJet

### DISCUSSION THREAD

**Discussion by lukasugar** [ToolJet/ToolJet#5422] [Q&A]
I'm already using ToolJet to classify some images. I'd like to expand it so that user can draw bounding boxes on the image.

Is there a way to draw bounding boxes on image in tooljet?

If there aren't any out of the box solutions, are there some javascript (preferably) or python libraries that can be used for this? Maybe some examles I could use?

Thanks!

**Comment by withshubh** [comment]
Hey @lukasugar - The bounding box component is in the roadmap of the ToolJet. We will update you as soon as we release it.
Please feel free to join [Community Slack](https://tooljet.com/slack) if you have more questions.

**Reply by lukasugar** [reply]
Thanks, that’s great to hear! Is there an ETA?
